### PR TITLE
add setup command

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: src/TobiasOlry/TalklyBundle/Command/SetupCommand.php [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+            //$bar->setBarWidth(100);
+
+            for ($i = 0; $i < 10; $i++) {
+                $bar->advance();
+
+                if (!$process->isRunning()) {
+                    break 2;
+                }
+
+                usleep(mt_rand(10000, 500000));
```
Comment (tolry) [inline src/TobiasOlry/TalklyBundle/Command/SetupCommand.php]: yeah: progress bars vs. reality - nice ;-)

File: src/TobiasOlry/TalklyBundle/Command/UserCreateCommand.php [hunk 2]
```diff
@@ -29,8 +34,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
 
-        $username = $io->ask('username');
-        $password = $io->askHidden('password');
+        if (!$username = $input->getArgument('username')) {
```
Comment (tolry) [inline src/TobiasOlry/TalklyBundle/Command/UserCreateCommand.php]: I thought this should normally be done in the `interact()` method (see http://symfony.com/doc/current/components/console/introduction.html#command-lifecycle)

File: src/TobiasOlry/TalklyBundle/Command/SetupCommand.php [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+        return $io->choice('Environment?', $choices, self::ENV_PROD);
+    }
+
+    /**
+     * @param OutputStyle $io
+     * @param string $env
+     */
+    private function setupDatabase(OutputStyle $io, $env)
+    {
+        $io->section('Setup database');
```
Comment (tolry) [inline src/TobiasOlry/TalklyBundle/Command/SetupCommand.php]: I would suggest `setting up database` or `database setup`

File: src/TobiasOlry/TalklyBundle/Command/SetupCommand.php [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+        $dbal = $this->getContainer()->get('doctrine.dbal.default_connection');
+
+        try {
+            $dbal->connect();
+        } catch (\Exception $e) {
+            throw new \RuntimeException('Database Server Connection fail! Please check your app/config/parameters.yml');
```
Comment (tolry) [inline src/TobiasOlry/TalklyBundle/Command/SetupCommand.php]: fail => failed

File: src/TobiasOlry/TalklyBundle/Command/SetupCommand.php [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+    }
+
+    /**
+     * @param OutputStyle $io
+     */
+    private function setupUser(OutputStyle $io)
+    {
+        $io->section('Create user');
+
+        if (!$io->confirm('do you want create an user?')) {
```
Comment (tolry) [inline src/TobiasOlry/TalklyBundle/Command/SetupCommand.php]: an => a

**Comment by tolry** [comment]
@DavidBadura  looks great

File: src/TobiasOlry/TalklyBundle/Command/UserCreateCommand.php (hunk 2 above)
Comment (DavidBadura) [inline src/TobiasOlry/TalklyBundle/Command/UserCreateCommand.php]: ahhh, nice :-)

**Pull request merged**